cookbook/security/multiple_guard_authenticators.rst

```diff
@@ -0,0 +1,168 @@
+How to Use Multiple Guard Authenticators
+========================================
+
+Guard authentication component allows you to easily use many different authenticators at a time.
+
+An entry point is a service id (of one of your authenticators) whose start()
+method should be called when an anonymous user hits a page that requires authentication.
+
+Multiple authenticators with shared entry point
```

Comment:
due to how we capitalise headlines this should be "Multiple Authenticators with a Shared Entry Point"